test/util-test.js

```diff
@@ -434,4 +434,41 @@ describe('util.addFormatMeta()', () => {
       });
     });
   });
+  describe('util.cutAfterJSON()', () => {
+    it('works with simple JSON', () => {
+      assert.equal(util.cutAfterJSON('{"a": 1, "b": 1}'), '{"a": 1, "b": 1}');
+    });
+    it('Cut extra characters after JSON  end', () => {
+      assert.equal(util.cutAfterJSON('{"a": 1, "b": 1}abcd'), '{"a": 1, "b": 1}');
+    });
+    it('tolerant to string constants', () => {
+      assert.equal(util.cutAfterJSON('{"a": "}1", "b": 1}abcd'), '{"a": "}1", "b": 1}');
+    });
+    it('tolerant to string with escaped quoting', () => {
+      // eslint-disable-next-line no-useless-escape
+      assert.equal(util.cutAfterJSON('{"a": "\\\"}1", "b": 1}abcd'), '{"a": "\\\"}1", "b": 1}');
+    });
+    it('works with nested', () => {
+      assert.equal(
+        // eslint-disable-next-line no-useless-escape
+        util.cutAfterJSON('{"a": "\\\"1", "b": 1, "c": {"test": 1}}abcd'),
+        // eslint-disable-next-line no-useless-escape, comma-dangle
```

Comment:
hmm, I don't know about disabling `comma-dangle`. I'll look into disabling this for function calls if it becomes too annoying